### PR TITLE
Adding use case for managing source swapping

### PR DESCRIPTION
**Pull request by Lletnek**
Adding use case as requested. For reference: http://discourse.wicg.io/t/need-loadstart-event-on-picture-src-change/985

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -785,6 +785,23 @@
...
+      <section>
+        <h3>
+          Managing source swapping between breakpoints
+        </h3>
+        <p>
+          When a new image is loaded via &lt;source&gt; after hitting a new
+          breakpoint, there is a delay while the image is loaded. Any CSS which
+          matches the breakpoint applies itself. However, we're stuck with the
+          previous image while the new one loads. If we had a JavaScript event
+          fire when the new image is loaded, loadstart for example. The user
```
Comment (yoavweiss) [inline index.html]: You want to avoid talking about a solution here, just the problem you're trying to solve.

File: index.html [hunk 2]
```diff
@@ -785,6 +785,23 @@
...
+          could listen for this and manage with CSS.
+        </p>
+        <p>
+          This could also be used on pages utilising a lazy-load on images,
```
Comment (yoavweiss) [inline index.html]: Can you expand on that? I'm not sure what that means.

Comment (Lletnek) [inline index.html]: Hey Yoav, do you mean the lazy-load, or previous para?

I think I'm gonna take the lazy-load reference out, it's not really relevant and doesn't add anything.

File: index.html [hunk 3]
```diff
@@ -785,6 +785,19 @@
...
       </section>
+      <section>
+        <h3>
+          Managing source swapping between breakpoints
+        </h3>
+        <p>
+          When a new image is loaded via &lt;source&gt; after hitting a new
+          breakpoint, there is a delay while the image is loaded. Any CSS which
+          matches the breakpoint applies itself. However, we're stuck with the
+          previous image while the new one loads. This can result in blurry
```
Comment (yoavweiss) [inline index.html]: Can you `s/we're stuck with the previous image/the previous image still displays/`?

File: index.html [hunk 4]
```diff
@@ -785,6 +785,19 @@
...
+          imagery, or a larger image being scaled down in place of a smaller
+          one.
+        </p>
```
Comment (yoavweiss) [inline index.html:800]: Can you add a phrase with something like "Developers need means to control that flash of poor quality imagery in these cases"?

**Comment by yoavweiss** [comment]
Cool. Now, if you can just address https://github.com/ResponsiveImagesCG/ri-usecases/pull/58#discussion_r35743720 we're good to go :)

File: index.html (hunk 4 above)
Comment (Lletnek) [inline index.html:800]: Sorry dude, I missed this. On it

**Comment by yoavweiss** [comment]
LGTM

**Pull request merged**